glide-core/src/request_type.rs

```diff
@@ -157,22 +570,22 @@ impl From<::protobuf::EnumOrUnknown<ProtobufRequestType>> for RequestType {
             ProtobufRequestType::ClientUnblock => RequestType::ClientUnblock,
             ProtobufRequestType::ClientUnpause => RequestType::ClientUnpause,
             ProtobufRequestType::Expire => RequestType::Expire,
-            ProtobufRequestType::HashSet => RequestType::HashSet,
-            ProtobufRequestType::HashGet => RequestType::HashGet,
-            ProtobufRequestType::HashDel => RequestType::HashDel,
-            ProtobufRequestType::HashExists => RequestType::HashExists,
+            ProtobufRequestType::HSet => RequestType::HSet,
```

Comment:
I like these changes but we might need to split them into a separate PR due to how large this PR already is. Not a strong opinion though, and not a blocker.